src/api/projects/mutations/addTranslationKeyToProject/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import createTranslationKey, {
+  type AddTranslationKeyToProjectArgs
+} from '@models/TranslationKey/createTranslationKey'
```

Comment:
Should both the method and the file maybe be renamed to `addTranslationKeyToProject`?

Reply:
Definitely